src/archivematicaCommon/lib/databaseFunctions.py

```diff
@@ -31,11 +31,19 @@
 from django.db import close_old_connections
 from django.db.models import Q
 from django.utils import six, timezone
+from prometheus_client import Summary
 from main.models import Agent, Derivation, Event, File, FPCommandOutput, SIP
 
 LOGGER = logging.getLogger("archivematica.common")
 
 
+db_retry_time_summary = Summary(
```

Comment:
Is there a reason not to introduce a `metrics.py` module in `archivematicaCommon` like the other packages have?

Reply:
Yeah, that's a good point. I was worried it would be confusing to have two metrics modules in the path — I think the local one (i.e. in MCPClient/Server) would be used, but still there's potential for getting mixed up. Also there are only a couple metrics used in archivematica common.

I could add common_metrics or something though? Thoughts on that?

Reply:
Ah! I didn't consider the path sharing :grimacing: It just seemed very well abstracted in the other cases, but you're right the MCPServer/Client have a lot more metrics. :+1: on keeping it like this.

Reply:
You're right, the other places are a bit cleaner 😅